frameworks/hdfs/docker-client/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:16.04
+
+RUN apt-get -y update && \
+    apt-get -y upgrade && \
+    apt-get -y install wget software-properties-common
+
+RUN wget https://downloads.mesosphere.com/hdfs/assets/hadoop-2.6.0-cdh5.9.1-dcos.tar.gz && \
+    tar xf hadoop-2.6.0-cdh5.9.1-dcos.tar.gz && \
+    rm hadoop-2.6.0-cdh5.9.1-dcos.tar.gz
+
+RUN apt-get update && \
+    apt-get install -y default-jre && \
```

Comment:
Nit: You can club the install of `default-jre` and `dnsutils` with the install of `wget` on line 5.